README.md

```diff
@@ -122,33 +117,40 @@ with tf.train.MonitoredTrainingSession(checkpoint_dir=checkpoint_dir,
     mon_sess.run(train_op)
 ```
 
-To run on a machine with 4 GPUs:
+## Running the Horovod
 
-```bash
-$ mpirun -np 4 python train.py
-```
+Below are a couple of examples of commands to run the distributed training. See the [Running the Horovod](docs/running.md)
```

Comment:
The example commands below show how to run distributed training. See the [Running Horovod](docs/running.md)